src/inference/tests/functional/caching_test.cpp

```diff
@@ -54,6 +54,8 @@ static const std::vector<std::string> cacheFolders{
     std::string("testCache"),
 };
 
+std::string getTestCaseName(const testing::TestParamInfo<std::tuple<TestParam, std::string>>& obj);
```

Comment:
Why did you add declaration before definition?
Looks strange, I don't believe that compiler reports warning for this.

Reply:
actually compiler reports...

'static' should be used to mark function as local instead of forward declaration.

Reply:
@alalek  thx for advice

@ilyachur fixed